pentaho-requirejs-osgi-manager/core/impl/src/main/java/org/pentaho/requirejs/impl/RequireJsConfigManager.java

```diff
@@ -126,7 +127,8 @@ private String getCachedContextMapping( String baseUrl, String referer ) {
         List<IRequireJsPackageConfiguration> requireJsPackageConfigurations = this.packageConfigurationsTracker.getPackages();
 
         for ( IRequireJsPackageConfiguration requireJsPackage : requireJsPackageConfigurations ) {
-          if ( referer.contains( baseUrl + requireJsPackage.getWebRootPath() ) ) {
+          String webRootPath = requireJsPackage.getWebRootPath();
+          if ( !StringUtils.isEmpty( webRootPath ) && referer.contains( baseUrl + webRootPath ) ) {
```

Comment:
Do we really need to import `commons-lang` for a single `webRootPath != null && webRootPath.length != 0`?
(in reality the only existing implementation of `IRequireJsPackageConfiguration` never returns `null` for `getName()`, but no harm being extra-careful)